README.md

```diff
@@ -1308,44 +1308,58 @@ import javax.ws.rs.core.Response;
 import org.folio.rest.jaxrs.model.TenantAttributes;
 
 public class MyTenantAPI extends TenantAPI {
- @Override
-  public void postTenant(TenantAttributes ta, Map<String, String> headers,
-    Handler<AsyncResult<Response>> hndlr, Context cntxt) {
+  @Validate
+  @Override
+  public void postTenant(TenantAttributes tenantAttributes, Map<String, String> headers,
+      Handler<AsyncResult<Response>> handler, Context context) {
 
     ..
-    }
+  }
+
+  @Validate
   @Override
-  public void getTenant(Map<String, String> map, Handler<AsyncResult<Response>> hndlr, Context cntxt) {
+  public void getTenant(Map<String, String> map, Handler<AsyncResult<Response>> handler, Context context) {
     ..
   }
   ..
 }
-
 ```
 
 If you wish to call the Post Tenant API (with Postgres) then just call the corresponding super-class, e.g.:
+
 ```java
+@Validate
 @Override
-public void postTenant(TenantAttributes ta, Map<String, String> headers,
-  Handler<AsyncResult<Response>> hndlr, Context cntxt) {
-  super.postTenant(ta, headers, hndlr, cntxt);
+public void postTenant(TenantAttributes tenantAttributes, Map<String, String> headers,
+  Handler<AsyncResult<Response>> handler, Context context) {
+  super.postTenant(tenantAttributes, headers, handler, context);
 }
 ```
-(not much point in that though - it would be the same as not defining it at all).
+
+(Not much point in that though - it would be the same as not defining it at all).
 
 If you wish to load data for your module, that should be done after the DB has been successfully initialized,
 e.g. do something like:
+
 ```
-public void postTenant(TenantAttributes ta, Map<String, String> headers,
-  super.postTenant(ta, headers, res -> {
-    if (res.failed()) {
-      hndlr.handle(res);
+public void postTenant(TenantAttributes tenantAttributes, Map<String, String> headers,
+  super.postTenant(tenantAttributes, headers, res -> {
```

Comment:
Missing a line above super.postTenant
`Handler<AsyncResult<Response>> handler, Context context) {`

Reply:
`res -> { ... }` is `Handler<AsyncResult<Response>> handler`, 
`context` in line 1362 is `Context context`.

Reply:
Thanks, good catch, fixed.